prep-javascript-arrays/main.js

```diff
@@ -0,0 +1,11 @@
+const languages = ['html', 'css', 'javascript', 'react', 'node', 'postgres'];
+console.log('languages array');
+console.log(languages);
+const firstElement = languages[0];
+console.log('the first element of the array is ' + firstElement);
+const thirdElement = languages[2];
+console.log('the third element of the array is ' + thirdElement);
+const length = languages.length;
+console.log('the length of the array is ' + length);
+const lastElement = languages[5];
```

Comment:
this should be done by calculation using the `length` variable you just created. 

The goal is to use that length value to calculate the last index of the array instead of directly targeting the element directly